app/controllers/agenda/agendaitems/agendaitem/index.js

```diff
@@ -78,7 +78,11 @@ export default class IndexAgendaitemAgendaitemsAgendaController extends Controll
       this.currentSession.may('manage-agendaitems'),
     );
   }
-
+  @action 
+  refresh() {
+    this.router.refresh(this.router.currentRouteName);
```

Comment:
The story specified explicitly to not refresh the page... I'm not sure this is the right way to do this